liquid-rust-syntax/src/grammar.lalrpop

```diff
@@ -138,7 +138,8 @@ BinOp5: ast::BinOp = {
 
 BinOp6: ast::BinOp = {
     "+" => ast::BinOp::Add,
-    "-" => ast::BinOp::Sub
+    "-" => ast::BinOp::Sub,
+    "%" => ast::BinOp::Mod,
```

Comment:
This is more like a mental note, no need to address it now as we can change it at any point:

Maybe the syntax in the logic should be `mod` instead of `%` to stress the semantics difference between both.

Reply:
Agreed. Eventually perhaps we can even generate a configurable compiler warning if used on i32s and not usizes.